src/main/resources/jenkins/plugins/office365connector/Webhook/config.jelly

```diff
@@ -0,0 +1,65 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry title="URL" field="url">
+        <f:textbox />
+    </f:entry>
+    <f:entry title="Name" field="name">
+        <f:textbox />
+    </f:entry>
+
+    <f:advanced>
+        <f:section title="Build status">
+            <f:description>
+                This section defines for which build statuses the notification is sent.
+            </f:description>
+
+            <f:entry title="Notify Build Start" field="startNotification">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Aborted" field="notifyAborted">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Failure" field="notifyFailure">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Not Built" field="notifyNotBuilt">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Success" field="notifySuccess">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Unstable" field="notifyUnstable">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Back To Normal" field="notifyBackToNormal">
+                <f:checkbox />
+            </f:entry>
+
+            <f:entry title="Notify Repeated Failure" field="notifyRepeatedFailure">
+                <f:checkbox />
+            </f:entry>
+        </f:section>
+
+        <f:section title="Advanced configuration">
+            <f:entry title="Macros" field="macros">
+                <f:repeatableProperty minimum="0" field="macros" add="${%Add Macro}">
```

Comment:
Macro got it's own config.jelly and made describable 👍

Reply:
minimum 0 again?

Reply:
I'll just use the same reply as below...

With zero removed: 
```
properties([office365Webhooks([[macros: [[]], url: '']])])
```

![image](https://user-images.githubusercontent.com/1661688/31359529-9f850bf8-ad4a-11e7-8a10-3de555b88a7e.png)


With zero added:
```
properties([office365Webhooks([[url: '']])])
```
![image](https://user-images.githubusercontent.com/1661688/31359434-47913660-ad4a-11e7-8b32-e9c98a1382d8.png)